app.config.ts

```diff
@@ -2,7 +2,7 @@ export default defineAppConfig({
   general: {
     title: "Nuxt Starters",
     logo: {
-      mainLogo: "",
+      mainLogo: "/dark-logo.png",
```

Comment:
How I decided this config would work is that if you want to switch between light and dark mode logos you leave the mainLogo empty. So mainLogo should be empty in this situation. Did you run into a problem with this ?

Reply:
Yeah, the issue is that when you load the site for the first time there is no logo 
![image](https://github.com/user-attachments/assets/92a9a536-9c9a-4029-956a-c4bcd097d669)
![image](https://github.com/user-attachments/assets/99baf9b0-0630-4aa2-8b66-4b5abe478e5d)

The logo will be there only when you switch themes.

 The other solution is to set a default theme in local storage